src/bot.js

```diff
@@ -44,6 +44,18 @@ const generateRandomColor = () => {
  */
 const fetchTodaysAnime = (channel, args) => {
   const TIMEZONE_INDEX = 0;
+  //console.log(args[1]);
+
+  const FORMAT=args[1];
+  var momentFormat;
+
+  if(FORMAT=="DMY"){
+    momentFormat="DD/MM/YYYY";
+  }else if(FORMAT=="MDY"){
+    momentFormat="MM/DD/YYYY";
+  }else if(FORMAT="YMD"){
+    momentFormat="YYYY/MM/DD";
+  }
```

Comment:
Since providing a date format is optional, you should try to add logic to handle the case when no format is provided. 

For example:
```
-> update  10000 America/Los_Angeles
```

Should update every 10 seconds with some default date format (YYYY/MM/DD ?). Otherwise, the timestamp is not very human readable (ex: Anime for 2018-07-08T21:30:50-07:00).